maps/_hghi-vents/scripts/cibuild

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${COVID19_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Build the site!
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        npm install
+        npm run build
+
+        # Synchronize everything to the Jekyll root
+        rsync -avz \
```

Comment:
The root `cibuild` script will always run the project level build process first. This makes sure that `_site` is cleared out, then populated again by the project level site components. When map specific `cibuild` scripts are run, the expectation is that they will handle copying the static bundle generated by their build processes into the appropriate location in `_site` (e.g., `maps/hghi-vents/`).